libs/common/src/lib/tasks-automation/tasks-automation.service.ts

```diff
@@ -40,6 +43,31 @@ export class TasksAutomationService {
     );
   }
 
+  duplicateTask(taskId: string) {
+    return this.modalService.openModal(TaskDuplicateDialogComponent).onClose.pipe(
+      filter((name) => !!name),
+      switchMap((name) =>
+        this.configs.pipe(
+          take(1),
+          map((configs) => configs.find((task) => task.id === taskId)),
+          filter((task) => !!task),
+          switchMap((task) => {
+            const parameters = task.parameters;
+            parameters.name = name;
+            parameters.operations = parameters.operations?.map((operation) => {
+              if (operation.ask) {
+                operation.ask.destination = `${operation.ask.destination}_copy`;
```

Comment:
so if you duplicate a duplicate, the destination name will be `destination_copy_copy`?

Reply:
yes